Automattic/wp-calypso
# Stepper: Leave checkout bring you to site-setup in onboarding

Fixes https://github.com/Automattic/wp-calypso/issues/97178

## Proposed Changes

We [fixed](https://github.com/Automattic/wp-calypso/pull/92168/files) leaving checkout for Start in the past, but not for Onboarding in Stepper.

When an user exits the checkout, we want to proceed as if the user has completed it (we'll propose again to pay in Launchpad or in /plans)

## Testing Instructions



- Go through `/setup/onboarding`
- Leave the checkout, emptying the cart
- You should land in /site-setup
- This should't change Start Onboarding
